libnmstate/ifaces/veth.py

```diff
@@ -38,10 +39,14 @@ def peer(self):
     def is_peer(self):
         return self.raw.get(_IS_PEER_METADATA) is True
 
+    @property
+    def is_peer_changed(self):
+        return self.raw.get(_PEER_CHANGED_METADATA) is True
```

Comment:
In stead of using `_PEER_CHANGED_METADATA`, you may just set `self._peer_changed = False` in `__init__()` which save a constant.

Reply:
Done, thanks!